backend/app/assets/stylesheets/spree/backend/components/_switch_solidus_admin.scss

```diff
@@ -0,0 +1,48 @@
+@import "spree/backend/themes/solidus_admin/colors";
+
+.switch {
```

Comment:
This can maybe live inside the solidus_admin theme and have a somewhat more specific class name, or keep the same name but scope it to the `.solidus-admin--nav` so we're sure not to have conflicts.